modules/nf-scil/registration/antsapplytransforms/main.nf

```diff
@@ -0,0 +1,63 @@
+process REGISTRATION_ANTSAPPLYTRANSFORMS {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+    tuple val(meta), path(image), path(reference), path(transform)
+
+    output:
+    tuple val(meta), path("*__warped.nii.gz"), emit: warpedimage
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def dimensionality = task.ext.dimensionality ? "-d " + task.ext.dimensionality : ""
+    def image_type = task.ext.image_type ? "-e " + task.ext.image_type : ""
+    def interpolation = task.ext.interpolation ? "-n " + task.ext.interpolation : ""
+    def output_dtype = task.ext.output_dtype ? "-u " + task.ext.output_dtype : ""
+    def default_val = task.ext.default_val ? "-f " + task.ext.default_val : ""
+
+    """
+    antsApplyTransforms $dimensionality\
+                        -i $image\
+                        -r $reference\
+                        -o ${prefix}__warped.nii.gz\
+                        $interpolation\
+                        -t $transform\
```

Comment:
Add the task.ext that you forgot to your command line.

I think it would be better to put options like $interpolation after the required input (-d -i -r -t -o).